### PR TITLE
adds CircleCI build which tests local build with GI suite

### PR DESCRIPTION
**Pull request by simpixelated**
This new circleci/config.yml will:
- checkout the branch and build the plugin
- setup a local docker network with MySQL, WordPress, and wp-cli
- install WordPress
- install plugins/config to make navigation work with ngrok
- install the locally built version of our plugin
- trigger our [Ghost Inspector tests](https://app.ghostinspector.com/tests/5cd1d1e08a512d55d82c83f0) which update the settings and verify it's correctly pulling data from the API

Thanks to @aaronfay for all the help with various Docker issues that came up!

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                --admin_password=admin \
+                --admin_email=foo@bar.com
+      - run:
+          name: Install Relative URL Plugin
+          command: |
+            docker run -it --rm \
+              --volumes-from wp-container \
+              --user xfs \
+              --network wp-network \
+              wordpress:cli plugin install relative-url \
```
Comment (aaronfay) [inline .circleci/config.yml]: That's cool.

**Review by aaronfay — APPROVED** [review]
This is a pretty awesome setup! I am confused about the tests though, it seems like the suite that is run is different than the suite that is displayed in Wordpress? The name is the same but in the admin I see 1 tests but Wordpress shows 2? What am I missing here?

**Comment by simpixelated** [comment]
@aaronfay I only wrote the test to check that at least one of the tests is there. It doesn't verify that every test in the suite shows up. That's something I could add to the GI test, but I'll have to figure out a better way to handle that as the number of tests grows.

**Comment by simpixelated** [comment]
Also I was moving tests around, renaming them, splitting them up into folders/suites, so there might be some inconsistencies, but I've settled on the one test for development now.

**Review by justinklemm — APPROVED** [review]
👏👏👏This is rad. I'm glad WordPress finally got a CLI!

**Pull request merged**